web_action_conditionable/static/src/js/views.js

```diff
@@ -1,22 +1,21 @@
-/*global openerp, _, $ */
-
-openerp.web_action_conditionable = function (instance) {
-  instance.web.View.include({
-    is_action_enabled: function(action) {
-      var attrs = this.fields_view.arch.attrs;
-      if (action in attrs) {
-	try {
-	  return this._super(action);
```

Comment:
Why did you remove this part? If `this._super` does something, we want to follow that first, no? Or is there no super?